terra/asset.json

```
@@ -1999,10 +1999,12 @@
   },
   {
     "id": "terra1gwrz9xzhqsygyr5asrgyq3pu0ewpn00mv2zenu86yvx2nlwpe8lqppv584",
+    "entity": "BLUE CUB DAO",
```

```suggestion
    "entity": "Lion Cub DAO",
```

@lioncubdao thanks for getting back on this so quickly, I just have one last question to clarify before merging the PR -- the `entity` field is mainly to "tag" a token to a specific project, and in this case, I'm assuming that the `BLUE` token belongs to your team, as such, would you like to update the entity to `Lion Cub DAO`? By doing so, the token would also show the associated telegram and twitter links.

See [`terra/entity.json`](https://github.com/coinhall/yacar/blob/b6e67dc004539353ca7b8126e64e5110f9493250/terra/entity.json#L150-L154)